packages/sanity/src/core/form/studio/tree-editing/components/breadcrumbs/TreeEditingBreadcrumbs.tsx

```diff
@@ -76,15 +75,16 @@ export function TreeEditingBreadcrumbs(props: TreeEditingBreadcrumbsProps): JSX.
   const [rootElement, setRootElement] = useState<HTMLDivElement | null>(null)
   const size = useElementSize(rootElement)
 
-  // todo: evaluate if these values are working as expected
-  // in various screen sizes
+  // a dropdown will show the "overflow items" when there are too many
+  // levels in the breadcrumbs. This will return how many
+  // items the breadcrumb allows before it starts to overflow into a new "..." item
+  // which keeps the rest
   const maxLength = useMemo(() => {
     const w = size?.border.width
 
     if (!w) return MAX_LENGTH
-    if (w < 300) return 3
-    if (w < 500) return 4
-    if (w < 700) return 5
+    if (w < 500) return 3
+    if (w < 700) return 4
```

Comment:
I don't know if this is the right number of breakpoints, while testing it felt right